ansible/idr-s3gateway.yml

```diff
@@ -28,17 +28,41 @@
       nginx_proxy_http2: true
       nginx_proxy_force_ssl: true
       # nginx_proxy_hsts_age: 31536000
-      nginx_proxy_backends:
-        - name: s3gateway
-          location: "~ ^/{{ idr_minio_s3_gateway_bucket | default('example') }}(/.*)?$"
-          server: "http://localhost:9000"
-      # Disable buffering and allow unlimited file sizes
-      # https://serverfault.com/a/818090
-      nginx_proxy_additional_directives:
-        - client_max_body_size 0
-        - proxy_http_version 1.1
-        - proxy_request_buffering off
-        - proxy_buffering off
+      nginx_proxy_sites:
+        # idr-ftp.openmicroscopy.org: S3 read-write /idr-upload only
+        - nginx_proxy_server_name: idr-ftp.openmicroscopy.org
+          nginx_proxy_backends:
+            - name: s3gateway
+              location: "~ ^/{{ idr_minio_s3_gateway_bucket | default('example') }}(/.*)?$"
+              server: "http://localhost:9000"
```

Comment:
Also not overly specific to this PR, but having the bucket as a variable but not the server seems somewhat odd.

Reply:
It's probably a bit confusing, because its an Nginx proxy to the S3 minio proxy to the real s3 embassy server

Reply:
I don't think that's what triggered me, but more it's unclear what gets encoded directly in the file and what is a variable.